### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by fluiddot**
Add `README` file for outlining how this repository works.

### CODE REVIEW

**Comment by fluiddot** [comment]
@oguzkocer I created this PR as a draft version of the `README` file, I haven't included the part related to publishing the artifacts to S3 as it's not defined yet.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+**Example:**
+`"react-native-get-random-values": "1.4.0"`
+2. Add the Android project of the dependency to the `settings.gradle.kts` file.
+
+**Example:**
+```
+include(":react-native-get-random-values")
+project(":react-native-get-random-values").projectDir = File(rootProject.projectDir, "node_modules/react-native-get-random-values/android")
+```
+3. Run command `npm install` to install the new pacakge.
```
Comment (oguzkocer) [inline README.md]:
```suggestion
3. Run command `npm install` to install the new package.
```

**Review by oguzkocer — COMMENTED** [review]
@fluiddot Thanks for doing this. I left a comment about a spelling error. Other than that, I think we might want to mention that not all projects would work with this setup, and some modifications might need to be made per project.

Also, how do you feel about mentioning something about asking a review from the #platform9 team for the PRs, just to be on the safe side that we don't accidentally break it - or make changes that'll get difficult to maintain in the long run?

**Comment by fluiddot** [comment]
> @fluiddot Thanks for doing this. I left a comment about a spelling error.

Thanks for addressing it 🙇 .

> Other than that, I think we might want to mention that not all projects would work with this setup, and some modifications might need to be made per project.

Good point, I'll add a note about what kind of dependencies are supported by this repository 👍 .

> Also, how do you feel about mentioning something about asking a review from the #platform9 team for the PRs, just to be on the safe side that we don't accidentally break it - or make changes that'll get difficult to maintain in the long run?

Ok, makes sense to me, we might even consider adding some people from #platofrm9 as [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), wdyt?

**Review by oguzkocer — APPROVED** [review]
Looks great! Thanks again @fluiddot 🙇‍♂️

**Pull request merged**